tests/plugins/print/print.js

```diff
@@ -0,0 +1,68 @@
+/* bender-tags: editor */
+/* bender-ckeditor-plugins: print */
+
+bender.editor = {
+	startupData: '<p>Foo</p>'
+};
+
+bender.test( {
+	// (#3661)
+	'test print command uses CKEDITOR.plugins.preview#createPreview()': function() {
+		var editor = this.editor,
+			createPreviewStub = sinon.stub( CKEDITOR.plugins.preview, 'createPreview', function() {
+				return {
+					$: {
+						print: function() {},
+						close: function() {},
+						document: {
+							execCommand: function() {}
+						}
+					}
+				};
+			} );
+
+		editor.once( 'afterCommandExec', function() {
+			resume( function() {
+				createPreviewStub.restore();
+
+				assert.areSame( 1, createPreviewStub.callCount );
+			} );
+		} );
+
+		editor.execCommand( 'print' );
+		wait();
+	},
+
+	// (#3661)
+	'test returning false from CKEDITOR.plugins.preview#createPreview cancels printing': function() {
+		var editor = this.editor,
+			createPreviewStub = sinon.stub( CKEDITOR.plugins.preview, 'createPreview', function() {
+				return false;
+			} );
+
+		editor.once( 'afterCommandExec', function() {
+			resume( function() {
+				createPreviewStub.restore();
+
+				assert.pass();
+			} );
+		} );
+
+		editor.execCommand( 'print' );
+		wait();
+	},
```

Comment:
Now I see why this test passes, as implementation checks against `false.$` which returns `undefined`. Although it's a bit hacky as it's rather expected that such code should throw an error as boolean is a primitive type. It seems a bit like leveraging language bad design which makes code less readable. See https://github.com/ckeditor/ckeditor4/pull/3672#discussion_r354867438

Reply:
TBH it was totally unintended and I didn't plan to use `false.$`.

At the same time this unit test doesn't seem to check anything sensible 🤔 it probably should check if print native command was invoked.